azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -924,9 +924,8 @@ private void ajaxExecuteFlow(final HttpServletRequest req,
       return;
     }
 
-    final ExecutableFlow exflow = new ExecutableFlow(project, flow);
+    final ExecutableFlow exflow = FlowUtils.createExecutableFlow(project, flow);
```

Comment:
I remember we have a method with the same name in test module. May we change the method name there, like `createTestExecutableFlow`?

azkaban.utils.TestUtils#createExecutableFlow

Reply:
:+1: